Resources/Private/Templates/Search/Main.html

```diff
@@ -12,15 +12,15 @@
 <html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
       data-namespace-typo3-fluid="true">
 
-<f:form action="search" controller="Search" name="searchParameter" method="get">
-    <label for="tx-dlf-search-query">
+<f:form action="search" controller="Search" name="searchParameter" method="get" class="tx-dlf-search-form">
+    <label for="tx-dlf-search-query-{viewData.uniqueId}">
         <f:if condition="{lastSearch.query}">
             <f:then>{lastSearch.query -> f:format.htmlspecialchars()}</f:then>
             <f:else><f:translate key="search.query"/></f:else>
         </f:if>
     </label>
 
-    <f:form.textfield id="tx-dlf-search-query" property="query" value="{lastSearch.query}" />
+    <f:form.textfield id="tx-dlf-search-query-{viewData.uniqueId}" property="query" value="{lastSearch.query}" />
```

Comment:
Wouldn't this break the search suggester, because it uses `$("#tx-dlf-search-query")`? (Also, there's some CSS in slub_digitalcollections and slub_web_sachsendigital that relies on the ID.)

Reply:
You're totally right. I reverted my change and brought back the former warning, not to change the id ;-)